protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -428,6 +428,22 @@ service TradingDataService {
 
     // Protocol Upgrade status
     rpc GetProtocolUpgradeStatus(GetProtocolUpgradeStatusRequest) returns (GetProtocolUpgradeStatusResponse);
+
+    // -- Decentralized History --
+    // Decentralised History allows the data-node to reach out to peer nodes to fetch the most recent history, as well as
+    // older history if desired, such that it can quickly get itself up to the latest block height of the network and start
+    // to consume events for the latest block from the Vega core.
+    // See https://github.com/vegaprotocol/vega/tree/develop/datanode/dehistory/README.md for a full description of Decentralised History
```

Comment:
```suggestion
    // Decentralized History
    //
    // Decentralised History allows the data-node to reach out to peer nodes to fetch the most recent history, as well as
    // older history if desired, such that it can quickly get itself up to the latest block height of the network and start
    // to consume events for the latest block from the Vega core.
    // See https://github.com/vegaprotocol/vega/tree/develop/datanode/dehistory/README.md for a full description of Decentralised History
```

Reply:
The line break here means that "Decentralized History" works as a title in the docs, rather than just some text. It will make things look a lot nicer/more organised when generated